integration_tests/src/util/workflows.rs

```diff
@@ -209,6 +210,46 @@ pub struct PrecommitMetadata {
     pub commd: CompactCommD,
 }
 
+pub fn build_verifreg_event(
```

Comment:
These are good helpers, but I don't think this is the right file for them. Making a new file `util/events` would probably be better. Or they could possibly go in the `expect` file.

Reply:
Done.